### ISSUE TITLE
The properties `CreatedTimeUtc` and `EndTimeUtc` in model `JobResponse` are always null

### ISSUE THREAD

**Issue by xzsheng** [Azure/azure-iot-sdk-csharp#169]
The properties `CreatedTimeUtc` and `EndTimeUtc` in model `JobResponse` are always null, when it was returned from API calls `IQuery.GetNextAsJobResponseAsync` and `JobClient.GetJobAsync`.

We checked the IoTHub response by Fiddler, it looks IoTHub DID return those times like snapshot below:
```
[{
	"jobId": "jobTest636415893635913881",
	"queryCondition": "tags.testTag = 'oldTagValue'",
	"createdTime": "2017-09-21T03:16:03.0318127Z",
	"startTime": "2017-09-21T03:16:03.6213869",
	"endTime": "2017-09-21T03:16:04.9515077Z",
    ...
}]
```

But the field names are not matching the model: IoTHub used `startTime` and `endTime`, while SDK expected `startTimeUtc` and `endTimeUtc`. Please check the [code](https://github.com/Azure/azure-iot-sdk-csharp/blob/master/service/Microsoft.Azure.Devices/JobClient/JobResponse.cs#L37) and

**Comment by CIPop** [comment]
Thanks @xzsheng for reporting!
Please feel free to submit a PR to fix this issue. Adding a unit test would be great to avoid this in the future.

**Comment by CIPop** [comment]
Fixed by #196.

**Comment by CIPop** [comment]
Thanks @xzsheng for the fix!
